pkg/repositories/gormimpl/common.go

```diff
@@ -114,3 +142,39 @@ func applyScopedFilters(tx *gorm.DB, inlineFilters []common.InlineFilter, mapFil
 	}
 	return tx, nil
 }
+
+func getGroupByForNamedEntity(tableName string) string {
```

Comment:
do these methods need to belong in common if they're only used in one place?

Reply:
Maybe not. Seemed like all of the joins/groups were in this file either way. And Go's package mechanism will expose them at package scope no matter which file they live in. So it seemed like a good idea to keep this near the other Db helper functions.

Reply:
sure but i mean logically do they need to belong in common? i'd say if they're specific to one repo impl it makes sense for them to live in that corresponding file

Reply:
Makes sense. I'll move them!